gce/pom.xml

```diff
@@ -56,7 +56,7 @@ Copyright 2019 Google LLC
     <dependency>
       <groupId>javax.servlet</groupId>
       <artifactId>javax.servlet-api</artifactId>
-      <version>4.0.0</version>
+      <version>3.1.0</version>
```

Comment:
any reason for this downgrade?

Reply:
It should never have been upgraded.  Jetty only supports 3.1.0